app/src/main/java/com/onboarding/wheatherapp/utils/StringUtils.kt

```diff
@@ -0,0 +1,9 @@
+package com.onboarding.wheatherapp.utils
+
+object StringUtils {
```

Comment:
as this object contains more things than only strings should be called ConstantsUtils or something similar

Reply:
Fix this in your next PR